en/resources/template-engines.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Template Engines
+menu: resources
+lang: en
+redirect_from: "/resources/template-engines.html"
+---
+
+# Template engines out-of-the-box compatible with Express
+
+- **[Pug](https://github.com/pugjs/pug)**: Haml inspired template engine (formerly Jade)
+- **[Haml.js](https://github.com/tj/haml.js)**: Haml implementation
```

Comment:
Haml is not really an Express template engine, is it?  I know it's relevant, but should this be included here?

Reply:
At least this link has an express template section: https://github.com/tj/haml.js/blob/master/Readme.md#express